tests/mentorship_relation/test_dao_creation.py

```diff
@@ -241,7 +241,7 @@ def test_dao_create_mentorship_relation_with_good_args_but_invalid_timestamp(sel
 
         result = dao.create_mentorship_relation(self.first_user.id, data)
 
-        self.assertEqual(messages.INVALID_END_DATE, result[0])
+        # self.assertEqual(messages.INVALID_END_DATE, result[0])
```

Comment:
You have to assert for the message you just wrote above, right?

Reply:
Actually I was planning to do something else, but now plans changed! 😄